File: solutions/python/resistor-color-duo/1/resistor_color_duo.py
full_colors = ['black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet', 'grey', 'white']

def value(colors):
    first = full_colors.index(colors[0])
    second = full_colors.index(colors[1])
    return int(str(first) + str(second))

File: solutions/typescript/resistor-color-duo/1/resistor-color-duo.ts
export function decodedValue(colors: string[]): number {
  return COLORS.indexOf(colors[0]) * 10 + COLORS.indexOf(colors[1]);
}

export const COLORS: string[] = [
  'black', 
  'brown',
  'red',
  'orange',
  'yellow',
  'green',
  'blue',
  'violet',
  'grey',
  'white',
]
